src/views/NewTeam.vue

```diff
@@ -52,23 +53,15 @@ const newTeamComponent = defineComponent({
 
     const teamName = ref("");
 
-    const validateTeamName = () => {
-      return;
-    };
+    const teamSubmissionTriggered = async () => {
+      const record = await createTeam(store.getters.generateAxiosInstance, {
```

Comment:
This seems a little weird to me because you're bypassing the store in order to do an api request directly. imo calls to api functions should only happen in the store layer and views should only interact with the store.

Reply:
We want to do this for the store in order to avoid unnecessary API calls being routed through the store; most components will not need central store effects based on an API Request, so they can make the requests themselves.